build/README.md

```diff
@@ -5,7 +5,7 @@ Building Kubernetes is easy if you take advantage of the containerized build env
 ## Requirements
 
 1. Docker, using one of the two following configurations:
-  1. **Mac OS X** You can either use docker-machine or boot2docker. See installation instructions [here](https://docs.docker.com/installation/mac/).  
```

Comment:
Double spaces at the end of a line convert to a linebreak in markdown. These should be added back to give the "note" more visibility.